runtime/doc/lsp.txt

```diff
@@ -1876,7 +1876,7 @@ apply_text_document_edit({text_document_edit}, {index}, {offset_encoding})
       • {text_document_edit}  (`lsp.TextDocumentEdit`)
       • {index}               (`integer?`) Optional index of the edit, if from
                               a list of edits (or nil, if not from a list)
-      • {offset_encoding}     (`string?`)
+      • {offset_encoding}     (`'utf-8'|'utf-16'|'utf-32'?`)
```

Comment:
Maybe it would be a good idea to create a separate `Encoding` enum type?

Reply:
There is an LSP type, but we don't render those into the docs so it opted to inline the values.